client/session.go

```diff
@@ -1174,8 +1174,6 @@ func (s *session) streamCollectedBlocksMetadata(
 		}
 		if received.submitted {
 			// Already submitted to enqueue channel
```

Comment:
This can definitely happen when there is a node being added to the cluster.  i.e. you expect two results back but since a new node is bootstrapping a third shard you will get an answer back from them too.  This is expected behavior.